docs/html/development/getting-started.rst

```diff
@@ -49,18 +49,18 @@ pip's tests are written using the :pypi:`pytest` test framework, :pypi:`mock`
 and :pypi:`pretend`. :pypi:`tox` is used to automate the setup and execution of
 pip's tests.
 
+It is preferable to run the tests in **parallel** for better user experience during development,
```

Comment:
I feel it is unnecessary to emphasise the parallelism, but it is personal preference.

Reply:
I guess the parallelism emphasis is what is laid in the attached issue #7683 
Do provide any changes you feel would go well

Reply:
I suggest removing the emphasis, since it looks weird.

Reply:
Acknowledged, changes in new commit to reflect the same